gcc/config/riscv/riscv-builtins.cc

```diff
@@ -155,12 +158,15 @@ AVAIL (always,     (!0))
 
 /* Argument types.  */
 #define RISCV_ATYPE_VOID void_type_node
+#define RISCV_ATYPE_UQI unsigned_intQI_type_node
```

Comment:
Do we need all of these argument types? Need to ensure I'm not implementing what I'm not using.